scm_v3c/scm3c_hw_interface.c

```diff
@@ -974,15 +975,15 @@ void radio_init_tx(){
     set_LC_current(127);
     
     // Set LDO voltages for PA and LO
-    set_PA_supply(63);
-    set_LO_supply(127,0);
+    set_PA_supply(127);
+    set_LO_supply(63,0);
```

Comment:
Could you confirm this will not effect the project using IEEE802.15.4 radio? such freq_sweep_tx/rx/lc_count?

Reply:
I verified that `freq_sweep_rx` works, but I couldn't get `freq_sweep_tx` working, even if I check out the current version from the `develop` branch.  After two fine code iterations, SCuM starts printing to UART.